src/manual-multiple-soulmates/web/index.html

```diff
@@ -2,16 +2,16 @@
     <header class="adverts__header">
         <h1 class="adverts__title">
             <a class="adverts__logo blink" href="%%CLICK_URL_UNESC%%[%TitleURL%]" data-link-name="title" target="_top">
-                {{#svg}}marque{{/svg}}
-                {{#svg}}soulmates-logo{{/svg}}
-                <!-- guardian logo -->
+            {{#svg}}guardian-soulmates-logo{{/svg}}
             </a>
         </h1>
-        <div class="adverts__blurb">Meet someone <em>worth</em> meeting</div>
+        <h1 class="adverts_blurb">
```

Comment:
Do you need this `h1` here? Previously it was just a `div` and you seem to still have that below (but as a `span`)

Reply:
All done - changed back to a `div` and 20px margin-top added to style.